go.mod

```diff
@@ -22,3 +22,5 @@ require (
 	sigs.k8s.io/controller-tools v0.6.3-0.20210916130746-94401651a6c3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/openshift/api => github.com/kwoodson/api v0.0.0-20211129174952-7efd8fe80ce3
```

Comment:
This will be removed once openshift/api/pull/1045 merges. An update to the API will be required at that time.